codes/codes_test.go

```diff
@@ -91,3 +92,12 @@ func (s) TestUnmarshalJSON_MarshalUnmarshal(t *testing.T) {
 		}
 	}
 }
+
+func (s) TestUnmarshalJSON_InvalidIntegerCode(t *testing.T) {
+	const wantErr = "invalid code: 200" // for integer invalid code, expect integer value in error message
+
+	var got Code
+	if err := got.UnmarshalJSON([]byte("200")); !strings.Contains(err.Error(), wantErr) {
+		t.Errorf("got.UnmarshalJSON(200) != %v; wantErr: %v", err, wantErr)
```

Comment:
The error message should have a `=` instead of a `!=` because we are printing `got` before `want`.

Reply:
Good catch. Done